task/06-conditions-n-loops-tasks.js

```diff
@@ -270,7 +315,12 @@ function isCreditCardNumber(ccn) {
  *   165536 (1+6+5+5+3+6 = 26,  2+6 = 8) => 8
  */
 function getDigitalRoot(num) {
-    throw new Error('Not implemented');
+    let array = num.toString().split("").map(Number);
+    let sum = array.reduce((accumulator, currentValue) => accumulator + currentValue);
```

Comment:
JFYI: use braces in each `for`/`while`/`do ... while`/`if` operator or `callback`. You will make less mistakes.